scripts/mixins/families/antlion_ambush.lua

```diff
@@ -15,12 +15,21 @@ g_mixins.families.antlion_ambush = function(antlionAmbushMob)
     antlionAmbushMob:addListener("ENGAGE", "ANTLION_AMBUSH_ENGAGE", function(mob, target)
         mob:useMobAbility(278) -- Pit Ambush
         mob:setMobMod(xi.mobMod.NO_MOVE, 0)
+        mob:timer(2000, function(ant)
+            ant:hideName(false)
```

Comment:
this stuff already happens in the mobskill itself, and takes care of the timing aspect so the timer call isn't needed - you get problems when you launch these things before the skill animation finishes

https://github.com/LandSandBoat/server/blob/d9d3a7b6494d17f2178b140c9b0f6d9e983d7459/scripts/globals/mobskills/pit_ambush.lua#L32